package.json

```diff
@@ -7,6 +7,7 @@
     "@testing-library/jest-dom": "^5.17.0",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "axios": "^1.4.0",
     "prop-types": "^15.8.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
```

Comment:
![image](https://github.com/Akorede3133/bookstore/assets/93039359/9f22018a-4314-4992-a35d-afbf1ceb525f)

- [ ] It would be great if you fix the issues link in your readme file because it is broken 😨 Kindly fix it to make your readme descriptive and professional 😊